frontend/src/pages/RecipientRecord/pages/Profile.js

```diff
@@ -9,66 +9,70 @@ import './Profile.css';
 import FeatureFlag from '../../../components/FeatureFlag';
 import ClassReview from './components/ClassReview';
 import MonitoringReview from './components/MonitoringReview';
-import { GrantDataProvider } from './GrantDataContext';
+import { useGrantData } from './GrantDataContext';
+import { ClassGuidanceProvider } from './ClassGuidanceContext';
 
 export default function Profile({
   recipientSummary,
   regionId,
   recipientId,
 }) {
   const activeGrants = recipientSummary.grants.filter((grant) => grant.status === 'Active');
+  const { hasMonitoringData, hasClassData } = useGrantData();
 
   return (
     <>
       <Helmet>
         <title>Profile</title>
       </Helmet>
-      <GrantDataProvider>
+      <ClassGuidanceProvider>
         <div className="maxw-widescreen">
           <Grid row gap={4}>
             <Grid desktop={{ col: 3 }} tabletLg={{ col: 12 }}>
-              <RecipientSummary summary={recipientSummary} regionId={regionId} />
+              <RecipientSummary summary={recipientSummary} />
             </Grid>
             <Grid desktop={{ col: 9 }} tabletLg={{ col: 12 }}>
-              <RecipientLeadership recipientId={recipientId} regionId={regionId} />
+              <RecipientLeadership recipientId={recipientId} />
             </Grid>
             <Grid desktop={{ col: 12 }} tabletLg={{ col: 12 }}>
               <GrantList summary={recipientSummary} />
             </Grid>
-            <FeatureFlag flag="monitoring">
-              {activeGrants.map((grant) => (
-                <React.Fragment key={grant.number}>
-                  <Grid desktop={{ col: 12 }}>
-                    <h2 className="smart-hub-title-big-serif">
-                      Grant number
-                      {' '}
-                      {grant.number}
-                    </h2>
-                  </Grid>
-                  <Grid desktop={{ col: 6 }} tabletLg={{ col: 12 }}>
-                    <div>
-                      <ClassReview
-                        grantNumber={grant.number}
-                        regionId={regionId}
-                        recipientId={recipientId}
-                      />
-                    </div>
-                  </Grid>
-                  <Grid desktop={{ col: 6 }} tabletLg={{ col: 12 }}>
-                    <div>
-                      <MonitoringReview
-                        grantNumber={grant.number}
-                        regionId={regionId}
-                        recipientId={recipientId}
-                      />
-                    </div>
-                  </Grid>
-                </React.Fragment>
-              ))}
-            </FeatureFlag>
+            {hasMonitoringData() || hasClassData() ? (
```

Comment:
Not sure if I'm understanding, but it looks from my testing that since these start out false, the data is never fetched and these never update, therefore nothing is ever rendered. Could be just the seed data I guess

Reply:
Whoops, that was a quick last minute change that didn't go as planned. Fixed here: 50d72b6869213aa6bd1a7fd333c08650f5069daa